p2p/transport/webrtc/errors.go

```diff
@@ -0,0 +1,29 @@
+package libp2pwebrtc
+
+import (
+	"fmt"
+)
+
+var (
+	errDataChannelTimeout = errDatachannel("timed out waiting for datachannel", nil)
+)
+
+func errConnectionFailed(msg string, err error) error {
+	return fmt.Errorf("peerconnection failed: %s: %v", msg, err)
+}
+
+func errDatachannel(msg string, err error) error {
+	return fmt.Errorf("datachannel error: %s: %v", msg, err)
+}
+
+func errMultiaddr(msg string, err error) error {
+	return fmt.Errorf("bad multiaddr: %s: %v", msg, err)
+}
+
+func errNoise(msg string, err error) error {
+	return fmt.Errorf("webrtc noise error: %s: %v", msg, err)
+}
+
+func errInternal(msg string, err error) error {
+	return fmt.Errorf("webrtc internal error: %s: %v", msg, err)
+}
```

Comment:
I think I've asked this before: These error helper functions are not very helpful (and not idiomatic). Can we just use normal error wrapping.

Errors need to be wrapped using `%w`, not `%v`, by the way.